src/main/java/com/okta/sdk/clients/AuthApiClient.java

```diff
@@ -114,6 +115,19 @@ public AuthResult authenticate(String username, String password, String relaySta
     }
 
     /**
+     * Authenticate into organization.
+     *
+     * @param  token {@link String}                 activationToken to use for authentication.
+     * @return {@link AuthResult}                   Result of the authentication transaction.
+     * @throws IOException                          If an input or output exception occurred.
+     */
+    public AuthResult authenticate(String token) throws IOException {
+        Map<String, Object> params = new HashMap<String, Object>();
```

Comment:
+1 for changing the JDK version along with this change. Otherwise it would still take two additional commits to fix this. (or tweak the pom for 1.7 source/target, and rebase this on top)